plugins/woocommerce/src/Internal/Admin/ProductReviews/ReviewsListTable.php

```diff
@@ -569,6 +569,17 @@ protected function handle_row_actions( $item, $column_name, $primary ) : string
 			);
 		}
 
+		/**
+		 * Filters the action links displayed for each review in the Reviews list table.
+		 *
+		 * @param string[]   $actions An array of comment actions. Default actions include:
+		 *                            'Approve', 'Unapprove', 'Edit', 'Reply', 'Spam',
+		 *                            'Delete', and 'Trash'.
+		 * @param WP_Comment $item The comment object.
+	 	*/
+		
+		$actions = apply_filters( 'comment_row_actions', array_filter( $actions ), $item );
```

Comment:
@sackadelic This will affect both regular comments and WooCommerce reviews. Should we use a dedicated filter name for this case since it pertains specifically to WooCommerce product reviews? Or do you intend to impact both WordPress comments and WooCommerce product reviews?

```suggestion
		$actions = apply_filters( 'woocommerce_product_review_row_actions', array_filter( $actions ), $item );
```

Reply:
@chihsuan It's a great question - currently, I'm filtering `comment_row_actions`, which affects all comments, and then, running logic to see if the comment is a review or not and stopping code execution if the comment is not a review.

We could keep it comment_row_actions for uniformity across the board and allow developers to filter out comments based on the review comment type or not, **or** we would need to change the filter hooks on the product edit pages as well (in WP Admin, when reviews are active).

Personally, for consistency, I think keeping the comment_row_actions filter would be the easiest to maintain and perform logic, but I'm up for ideas.

Reply:
Thanks for the clarification. I can see that this `handle_row_actions` follows the same logic as WP core's `handle_row_actions`. It makes sense to keep the comment_row_actions filter for consistency!